UKHO.MaritimeSafetyInformation.Web/UKHO.MaritimeSafetyInformation.Web/wwwroot/js/Custom/RadioNavigationalWarnings-admin.js

```diff
@@ -31,3 +31,47 @@ function RnwFilterList(pageSelectedIndex) {
     rnwUrl = rnwUrl.replace("amp;", "");
     window.location.href = rnwUrl;
 }
+
+function do_Selection() {
+    var checkboxes = document.getElementsByName('checkbox');
+    var button = document.getElementById('select_button');
+
+    if (button.value == 'Select all') {
+        for (var i in checkboxes) {
+            checkboxes[i].checked = 'FALSE';
+            document.getElementById("BtnShowSelection").disabled = false;
+        }
+        button.value = 'Clear all'
+    } else {
+        for (var i in checkboxes) {
+            checkboxes[i].checked = '';
+            document.getElementById("BtnShowSelection").disabled = true;
```

Comment:
document.getElementById("BtnShowSelection").disabled = true;
This statement should be outside of for loop.

Reply:
done required changes